proto/viam/component/camera/v1/camera.proto

```diff
@@ -12,29 +12,47 @@ option java_package = "com.viam.component.camera.v1";
 
 // A CameraService services all cameras associated with a robot
 service CameraService {
-  // GetImage returns a frame from a camera of the underlying robot. A specific MIME type
-  // can be requested but may not necessarily be the same one returned.
+  // GetImage returns a frame from a camera of the underlying robot. A specific
+  // MIME type can be requested but may not necessarily be the same one
+  // returned.
   rpc GetImage(GetImageRequest) returns (GetImageResponse) {
+    option deprecated = true;
     option (google.api.http) = {get: "/viam/api/v1/component/camera/{name}/image"};
   }
 
   rpc GetImages(GetImagesRequest) returns (GetImagesResponse) {
+    option deprecated = true;
     option (google.api.http) = {get: "/viam/api/v1/component/camera/{name}/images"};
   }
 
-  // RenderFrame renders a frame from a camera of the underlying robot to an HTTP response. A specific MIME type
-  // can be requested but may not necessarily be the same one returned.
+  // RenderFrame renders a frame from a camera of the underlying robot to an
+  // HTTP response. A specific MIME type can be requested but may not
+  // necessarily be the same one returned.
   rpc RenderFrame(RenderFrameRequest) returns (google.api.HttpBody) {
+    option deprecated = true;
```

Comment:
Wondering why we needed this custom API endpoint for the frontend to begin with since the web app can handle GRPC responses.